### ISSUE TITLE
Adding Category,Tag page link

### ISSUE THREAD

**Issue by fukugit** [MeiK2333/github-style#15]
Hi,
Thank you for providing an awesome theme. I use it.

The menu bar in a header seems not to have a category, tag link that goes through to those pages.

That is my personal opinion.
If this layout has it, it will get more helpful for the user. 
So, I'd like you to add those links if you agree with that opinion.

I hope you will consider my request. Thank you.

**Comment by MeiK2333** [comment]
Good idea! but I've been busy recently. can you wait for a while? or you can commit a pull request.

**Comment by fukugit** [comment]
Actually, I tried to add the category page under the 'partials' folder but it didn't appear on the web page, and a link does not work too.
Do I need to do another setting? If I fix it, I'm going to send a pull request.

**Comment by winterant** [comment]
Is it supported now？
